### PR TITLE
Add doc about new SCM integrations

### LINKED ISSUE #651: Add documentation about new SCM integrations
GitLab, GitLab self hosted, GitHub Enterprise

### PR DESCRIPTION
**Pull request by EtienneM**
To be merged when the feature is in prod

Fix #651

### CODE REVIEW

File: _posts/languages/ruby/2000-01-01-getting-started-with-ruby.md [hunk 1]
```diff
@@ -158,6 +161,8 @@ You know how to deploy a Ruby app, now develop your project and build something
 
 Here are some links that you could find useful:
 
-* Need to store data? Look at our [database addons](https://scalingo.com/fr/databases)
+* Need to store data? Look at our [database
+  addons](https://scalingo.com/fr/databases)
```
Comment (yannski) [inline _posts/languages/ruby/2000-01-01-getting-started-with-ruby.md]:
```suggestion
  addons](https://scalingo.com/databases)
```

**Comment by yannski** [comment]
@EtienneM when doc page are deleted, a redirection should be added. Take a look at https://github.com/Scalingo/documentation/blob/master/redirections.yml.

**Comment by yannski** [comment]
BTW, for SEO purpose, I'm wondering if separate pages should be added "Github and GitHub Enterprise integration" and "GitLab integration". What do you think?

**Comment by EtienneM** [comment]
> BTW, for SEO purpose, I'm wondering if separate pages should be added "Github and GitHub Enterprise integration" and "GitLab integration". What do you think?

I have no knowledge on SEO. But aren't we punished if there is some duplicate content? And we'll have a lot of duplicate content if we have two pages for GitHub and GitLab.

**Comment by yannski** [comment]
I may be mistaken but the page will be quite different (even tough some wording will be the same)

**Pull request merged**